server/testutils/containers_e2e.go

```diff
@@ -0,0 +1,62 @@
+package testutils
```

Comment:
Here are the helper functions to initialize the environment for this plugin, this is specific to this plugin and uses the "testcontainer mattermost module" created. Also I added the function for mocking things using the client library.